.hack/update-artifacthub-pkg.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
The other script (`.hack/verify-files-structure.sh`) uses `#!/bin/bash`. Maybe change for consistency.